### PR TITLE
PostgreSQL driver support for JPA/Hibernate in native image

### PR DESCRIPTION
**Pull request by Tomas-Kraus**
To run the tests, build Helidon and then execute:
```
cd <helidon_root>/tests/integration/jpa
mvn clean install -Dpgsql -Pdocker -pl model,appl -Pnative-image -Dnative-image
```
```
Tests run: 16, Failures: 0, Errors: 0, Skipped: 0
```

### CODE REVIEW

**Comment by Tomas-Kraus** [comment]
First waiting for native image the tests to finish.

**Comment by Tomas-Kraus** [comment]
@barchetta This is currently blocked by PLS so adding you as reviewer. Please let us know whether it is mofing forward somehow.

**Comment by Tomas-Kraus** [comment]
@romain-grecourt One of the tests failed during native image build. This is quite strange, because it's not usually happening. I saw it for the first time here. Do you have any idea what's going wrong? Is it resources (memory, disk space) problem?

**Comment by Tomas-Kraus** [comment]
I had to remove parallel jobs from Jenkinsfile in master because of resource issues. This PR was rebased and squashed to resolve conflicts.

**Review by barchetta — APPROVED** [review]
Approving third party dependencies.

File: integrations/db/pgsql/pom.xml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+    <properties>
+        <spotbugs.skip>true</spotbugs.skip>
+        <maven.javadoc.skip>true</maven.javadoc.skip>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
```
Comment (barchetta) [inline integrations/db/pgsql/pom.xml:40]: Ooops. Spoke too soon. Please update dependencies/pom.xml to use version 42.2.18

**Review by barchetta — CHANGES_REQUESTED** [review]

File: integrations/db/pgsql/pom.xml (hunk 1 above)
Comment (Tomas-Kraus) [reply integrations/db/pgsql/pom.xml:40]:
```
[INFO] Copying postgresql-42.2.18.jar to /data/JPA_PGSQL_NI/tests/integration/jpa/appl/target/libs/postgresql-42.2.18.jar
```

**Review by barchetta — APPROVED** [review]

File: etc/scripts/test-integ-mysql.sh [hunk 2]
```diff
@@ -52,8 +52,12 @@ mvn ${MAVEN_ARGS} -f ${WS_DIR}/pom.xml \
 
 # Run tests in Java VM application
 (cd tests/integration/jpa && \
-  mvn ${MAVEN_ARGS} clean verify -Dmysql)
+  mvn ${MAVEN_ARGS} clean verify \
+      -Dmaven.test.failure.ignore=true -Dmysql \
```
Comment (tomas-langer) [inline etc/scripts/test-integ-mysql.sh:56]: This is wrong - this PR is for `psql` not for `mysql`?

File: etc/scripts/test-integ-pgsql.sh [hunk 3]
```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -ex
+#
+# Copyright (c) 2018, 2020 Oracle and/or its affiliates.
```
Comment (tomas-langer) [inline etc/scripts/test-integ-pgsql.sh:3]: Copyright is wrong (as with other new files in this PR) - the year should be just `2020`

**Review by tomas-langer — APPROVED** [review]
If the changes for mysql are required as part of this PR, it can be merged.

File: etc/scripts/test-integ-pgsql.sh (hunk 3 above)
Comment (Tomas-Kraus) [reply etc/scripts/test-integ-pgsql.sh:3]: Yes, it's copy-paste. I'll fix it in next PR.

File: etc/scripts/test-integ-mysql.sh (hunk 2 above)
Comment (Tomas-Kraus) [reply etc/scripts/test-integ-mysql.sh:56]: There was -Dmaven.test.failure.ignore=true  missing so I added it into both scripts. They are not being used now and we'll do another review after Romain prepares parallel jobs setup for all those native image integration tests.

**Pull request merged**